README.md

```diff
@@ -1,21 +1,46 @@
+
+Key security features: 
+
+- **Identity-less**: no need to disclose identity information (such as name, email address, or phone number) to be able to transfer files.
+- **End-to-end encryption**: files are end-to-end encrypted and only the sender and recipient can read them.
+- **Peer-to-peer** file sharing: Destiny attempts to make a direct network connection to the other party. If both parties are on the same local network they should connect without any traffic leaving that network, for example. When this isn’t possible (e.g. if neither party has a public IP address) then our relay server is used (however, that server sees only encrypted packets).
+- **Full-strength keys**: although our codes are short and human-memorable, they are part of an online “Password Authenticated Key Exchange” (PAKE) which only allows a single guess – and yields a 256-bit full-strength symmetric key.
+
+
+Based on the [wormhole-william](https://github.com/LeastAuthority/wormhole-william)
 implementation of the [Magic Wormhole protocol](https://github.com/magic-wormhole/magic-wormhole).
-This application relies on a plugin that wraps the wormhole-william client
-with a Dart API. The plugin is implemented [here](https://github.com/LeastAuthority/dart_wormhole_william).
+This application relies on [a plugin that wraps the wormhole-william client](https://github.com/LeastAuthority/dart_wormhole_william)
+with a Dart API.
 
 Supported platforms
 
-- [x] Linux
+- [x] Linux (amd64)
 - [x] Windows
-- [x] OSX
+- [x] macOS
 - [x] Android
 - [ ] iOS
 
+## Usage
+
+### Installation
+
+You can find detailed instructions on how to install applications on various platforms [here](https://github.com/LeastAuthority/destiny/blob/main/doc/installation.md)
+
+[Windows](https://github.com/LeastAuthority/destiny/releases/latest/download/destiny_windows.msix), [Linux](https://github.com/LeastAuthority/destiny/releases/latest/download/destiny_linux_amd64.AppImage), [macOS](https://github.com/LeastAuthority/destiny/releases/latest/download/destiny_macos.dmg) files can be downloaded from the latest [release](https://github.com/LeastAuthority/destiny/releases/latest).
+
+Android will be available in (Google Play store)[https://play.google.com/store/apps/details?id=com.leastauthority.destiny]. Alternatively can be downloaded manually from [here](https://github.com/LeastAuthority/destiny/releases/latest/download/destiny_android.apk)
+
+
+
+### Verification
+
+We recommend verifying every downloaded file. Instructions [here](https://github.com/LeastAuthority/destiny/blob/main/doc/releases.md). 
```

Comment:
Similar comment for this. Maybe "We recommend [verifying every downloaded file] against the corresponding signature".